### PR TITLE
Ch6 formula corrections

### PR DESCRIPTION
**Pull request by alv-around**
I found some minor errors in the formulas in Chapter 6 Statements. These are not critical, but they can cause unnecessary confusion.

For ease of comparison, for each of the changes I made, I attach a screenshot of the actual pdf online (linked in the [README.md](https://github.com/LeastAuthority/moonmath-manual)) and a screenshot of the pdf of the proposed changes.

### CODE REVIEW

File: chapters/statements-moonmath.tex [hunk 1]
```diff
@@ -1397,12 +1397,12 @@ \subsubsection{QAP Satisfiability} One of the major points of Quadratic Arithmet
...
+\begin{example} Consider the Quadratic Arithmetic Program $QAP(R_{3.fac\_zk})$ from \examplename{} \ref{ex:3-fac-QAP} and its associated R1CS from equation \ref{ex:3-factorization-r1cs}. To give an intuition of how proofs in the language $L_{QAP(R_{3.fac\_zk})}$ look like, lets consider the instance $I_1=11$. As we know from \examplename{} \ref{ex:3-fac-zk-circuit_2}, $(W_1,W_2,W_3,W_4)=(2,3,4,6)$ is a proper witness, since 
+$(<I_1>;<W_1,W_2,W_3,W_4>)=(<11>;<2,3,4,6>)$ is a valid circuit assignment and hence, a solution to $R_{3.fac\_zk}$ and a constructive proof for language $L_{R_{3.fac\_zk}}$. 
 
 In order to transform this constructive proof into a knowledge proof in language $L_{QAP(R_{3.fac\_zk})}$, a prover has to use the elements of the constructive proof, to compute the polynomial $P_{(I;W)}$. 
 
-In the case of $(<I_1>;<W_1,W_2,W_3,W_5>)=(<11>;<2,3,4,6>)$,  the associated proof is computed as follows: 
+In the case of $(<I_1>;<W_1,W_2,W_3,W_4>)=(<11>;<2,3,4,6>)$,  the associated proof is computed as follows:
```
Comment (alv-around) [inline chapters/statements-moonmath.tex:1405]: From example 120, the R1CS for 3-Factorization:

R1CS - 3-factorization

The constrain system has only 4 Witness values, but the QAP section, in `Example 132` some parts use $W_5$ which should be $W_4$ 

`actual`:

actual - example 132
actual - example 132  II

`proposed`:

proposed - example 132

**Review by alv-around — COMMENTED** [review]
Here each of the changes explained in detail

**Comment by PlanetMacro** [comment]
Thanks for noticing. We resolved it in another PR:

**Comment by alv-around** [comment]
> Thanks for noticing. We resolved it in another PR:

Hi @PlanetMacro, I just realized that there was a previous PR (#109) correcting the last typo (Line 1401 & 1405). However the first two typos (Line 135 & 241) are still open. If you want I can isolate them in a new PR.

File: chapters/statements-moonmath.tex (hunk 1 above)
Comment (alv-around) [reply chapters/statements-moonmath.tex:1405]: Already corrected in #109

**Pull request closed without merging**